apps/web/src/store/use-auth-controller.ts

```diff
@@ -64,6 +64,12 @@ export function useAuthController() {
     applyToken(token);
 
     if (token) {
+      queryClient.removeQueries({
+        predicate: (query) =>
+          query.queryKey !== '/v1/users/me' &&
+          query.queryKey !== '/v1/environments' &&
+          query.queryKey !== '/v1/organizations/me',
```

Comment:
Oh got ya, love it. But will also refetch them aswell?

Reply:
It will auto-refetch since react-query is designed to do it.